forem/forem
# Remove asset reconciliation

## What type of PR is this? (check all applicable)

## Description
This removes the `_asset_reconciliation.html.erb` file and its related code since it introduced other bugs while fixing some other ones. We want to take an approach where we're fixing the specific bugs that appeared before the `asset_reconciliation` hack.

## Related Tickets & Documents
See #12725 for more details

Resolves #12725...
and maybe #12631, #12455

## QA Instructions, Screenshots, Recordings
1. Before pulling this branch, update your user's theme to something non-default (like night theme)
2. Go to the home page
3. See that it has a flash of going from night theme to the default theme (user set theme not persisted)
4. Pull this branch down
5. Refresh (not hard refresh) the page
6. See that your theme persists

### UI accessibility concerns?
Nope

## Added tests?
- [x] No, and this is why: removing a feature

## [Forem core team only] How will this change be communicated?

## [optional] Are there any post deployment tasks we need to perform?
Nope

## [optional] What gif best describes this PR or how it makes you feel?

![Spongebob smiles with satisfaction, and brushes his hands like he's done.](https://media.giphy.com/media/26u4lOMA8JKSnL9Uk/giphy.gif)
